### DISCUSSION TITLE
[Feature] Angled hitboxes

### DISCUSSION THREAD

**Discussion by Amidweiz** [ikemen-engine/Ikemen-GO#894] [Feature Request]
I'm not sure if this would be able to be done but it would be neat if you where able to angle/rotate hitboxes rather than needing to angle hitboxes like this.
![hitbox](https://user-images.githubusercontent.com/55671166/153313961-ff1cd0e9-7939-4b3b-b64a-05e9c38d7034.png)
Not sure if this would need a ray of sorts or if it could be done by pure angle similar to how the angledraw sctrl works, however it would reduce hitbox complexity by quite a bit.

**Comment by potsmugen** [comment]
Already implemented with TransformClsn.
